model/patch/github.go

```diff
@@ -127,7 +130,8 @@ func NewGithubIntent(msgDeliveryID string, event *github.PullRequestEvent) (Inte
 		event.Sender == nil || event.Sender.Login == nil ||
 		event.PullRequest == nil || event.PullRequest.DiffURL == nil ||
 		event.PullRequest.Head == nil || event.PullRequest.Head.SHA == nil ||
-		event.PullRequest.Head.Repo == nil || event.PullRequest.Head.Repo.FullName == nil {
+		event.PullRequest.Head.Repo == nil || event.PullRequest.Head.Repo.FullName == nil ||
+		event.PullRequest.Title == nil {
```

Comment:
nit: I'd be inclined to move this validation into its own function for more clear testing